app/src/services/dataset.service.ts

```diff
@@ -0,0 +1,38 @@
+import { RWAPIMicroservice } from 'rw-api-microservice-node';
+import logger from 'logger';
+
+// @ts-ignore
+import DatasetNotFound from 'errors/datasetNotFound.error';
+
+const API_VERSION: string = 'v1';
```

Comment:
this is used once, [KISS](https://en.wikipedia.org/wiki/KISS_principle) says remove it